src/Components/Web.Extensions/src/wwwroot/headManager.js

```diff
@@ -0,0 +1,51 @@
+// Local helpers
+
+const getHeadElement = (tagName, id) => {
+    const elements = Array.from(document.getElementsByTagName(tagName));
+    return elements.find(e => e._blazorId === id);
+}
+
+const getOrCreateHeadElement = (tagName, id) => {
+    let element = getHeadElement(tagName, id);
+
+    if (element) {
+        return element;
+    }
+
+    element = document.createElement(tagName);
+    element._blazorId = id;
+
+    const head = document.getElementsByTagName('head')[0];
```

Comment:
```suggestion
    const head = document.head;
```

... unless you know of any reason why not :)